### ISSUE TITLE
Accessibility - Home page - Video iframes missing title

### ISSUE THREAD

**Issue by terrimorton** [dotnet-foundation/website#861]
This issue was found by running the Accessibility Insights for Web browser extension in Chrome.

URL: https://dotnetfoundation.org/

There are 2 instances of iframes on this page which do not have a title.

- <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/videoseries?list=PL1rZQsJPBU2S4_ZjpE20DJcPT8okkXPja" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen=""></iframe>
- <iframe id="projectYouTube" width="560" height="315" src="https://www.youtube.com/embed/c5VUoERRvbM" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe>

Fix ONE of the following:

- Element has no title attribute
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element's default semantics were not overridden with role="none" or role="presentation"

**Comment by Arhell** [comment]
@terrimorton done
